pkg/controllers/restore/controller.go

```diff
@@ -273,20 +275,33 @@ func (h *handler) OnRestoreChange(_ string, restore *v1.Restore) (*v1.Restore, e
 	return restore, err
 }
 
-func (h *handler) restoreCRDs(created map[string]bool, objFromBackupCR ObjectsFromBackupCR) error {
+func (h *handler) restoreCRDs(created map[string]bool, objFromBackupCR ObjectsFromBackupCR) ([]string, error) {
```

Comment:
I would do a named return for `crdsWithSubStatus` here for the documentation purpose

Reply:
fixed.